core/conf/demo/server_config.yaml

```diff
@@ -46,9 +46,12 @@ server_config:
 #                      | loaded when Milvus server starts up.                       |            |                 |
 #                      | '*' means preload all existing tables.                     |            |                 |
 #----------------------+------------------------------------------------------------+------------+-----------------+
+# auto_flush_interval  | Interval of auto flush. Unit is millisecond.               | Integer    | 1000 (ms)       |
```

Comment:
The interval, in milliseconds, at which Milvus automatically flushes data to disk.

Reply:
Please apply this change to other files.